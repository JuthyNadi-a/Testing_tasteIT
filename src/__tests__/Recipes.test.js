import React from "react";
import renderer from "react-test-renderer";

import Recipes from '../components/Recipes';

it('renders when there is no recipe', () => {
    const tree = renderer.create(<Recipes />).toJSON();
    expect(tree).toMatchSnapshot();
})
