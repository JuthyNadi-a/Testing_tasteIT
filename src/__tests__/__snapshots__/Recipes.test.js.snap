// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there is no recipe 1`] = `
<div
  className="mainContainer"
>
  <div
    className="searchBar"
  >
    <input
      onChange={[Function]}
      placeholder="Search by recipe name"
      string=""
      type="search"
    />
    <label
      className="search"
      htmlFor="search"
    >
      Search
    </label>
  </div>
  <div
    className="cards"
  />
</div>
`;
